controllers/users/_preview_toolbar.htm

```diff
@@ -17,6 +17,15 @@
         <?= e(trans('rainlab.user::lang.users.impersonate_user')) ?>
     </a>
 <?php endif ?>
+
+<a
+    href="javascript:;"
+    data-request="onUnsuspendUser"
+    data-request-confirm="<?= e(trans('rainlab.user::lang.users.unsuspend_confirm')) ?>"
+    class="btn btn-default oc-icon-undo">
+    <?= e(trans('rainlab.user::lang.users.unsuspend')) ?>
+</a>
```

Comment:
Can we have the button only show up if the user is actually suspended?

Reply:
The button element should be indented just like the one above.

Reply:
Missed that one when adding the conditional. Apologies, all sorted now.